### PR TITLE
Adding instructions for Linux simulator

### PR DESCRIPTION
**Pull request by wduraes**
Adding the Linux simulator README file

### CODE REVIEW

File: demos/projects/PC/linux/README.md [hunk 1]
```diff
@@ -1 +1,103 @@
...
+
+* [CMake](https://cmake.org/download/) (Version 3.20 or higher)
+
+* Execute the installation script 
+
+```bash
+sudo ./.github/scripts/install_software.sh
+```
+
+* Execute the Network setup script
```
Comment (hihigupt) [inline demos/projects/PC/linux/README.md:31]: Can we say that this script will create veth0 and veth1 interfaces.

File: demos/projects/PC/linux/README.md [hunk 2]
```diff
@@ -1 +1,103 @@
...
+*While this sample supports SAS keys and Certificates, this guide will refer only to SAS keys. 
+
+## Prepare the simulation
+
+To connect the simulated device to Azure, you'll modify a configuration file for Azure IoT settings, rebuild the image, and run it.
+
+Update the file `demo_config.h` with your configuration values.
+
+```bash
+sudo nano demos/projects/PC/linux/config/demo_config.h
```
Comment (hihigupt) [inline demos/projects/PC/linux/README.md:52]: no need to do sudo

nano demos/projects/PC/linux/config/demo_config.h

File: demos/projects/PC/linux/README.md [hunk 3]
```diff
@@ -1 +1,103 @@
...
+
+```bash
+    sudo tcpdump --list-interfaces
+```
+Look for line #138 and update configNETWORK_INTERFACE_TO_USE with the number you got in the previous step. 
+
+**Example**: if you got ``4.veth1 [Up, Running]`` in the previous step, you'll update line #138 to look like this ``#define configNETWORK_INTERFACE_TO_USE (4L)``
+
+```bash
+sudo nano demos/projects/PC/linux/config/FreeRTOSConfig.h
```
Comment (hihigupt) [inline demos/projects/PC/linux/README.md:83]: no need to do sudo here only do nano and file name

File: demos/projects/PC/linux/README.md (hunk 1 above)
Comment (wduraes) [reply demos/projects/PC/linux/README.md:31]: done!

File: demos/projects/PC/linux/README.md (hunk 2 above)
Comment (wduraes) [reply demos/projects/PC/linux/README.md:52]: thanks! old habits!!

File: demos/projects/PC/linux/README.md [hunk 4]
```diff
@@ -1 +1,103 @@
...
+
+```bash
+    cd iot-middleware-freertos-samples
+    git submodule update --init --recursive --depth 1
+```
+
+
+## Install Prerequisites
+
+* [CMake](https://cmake.org/download/) (Version 3.20 or higher)
```
Comment (hihigupt) [inline demos/projects/PC/linux/README.md:23]: our requirement is 3.13 and higher

**Review by hihigupt — APPROVED** [review]

File: demos/projects/PC/linux/README.md (hunk 4 above)
Comment (wduraes) [reply demos/projects/PC/linux/README.md]: got version 3.20 stuck in my mind for some reason...

**Pull request merged**